DotNetCorePlugins.sln

```diff
@@ -133,6 +127,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Mixer", "samples\dynamic-im
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ServiceImplementation", "samples\dynamic-implementation\ServiceImplementation\ServiceImplementation.csproj", "{39BCFE6F-50C4-45AE-8D13-38FB423AB619}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "JsonNet13", "test\TestProjects\JsonNet13\JsonNet13.csproj", "{9F7041CA-98DE-4DFD-8BFC-71B0338D4CB0}"

```

Comment:
How will this new test project be used? I did not see anything which references it, so I'm not sure if this is needed or not.

Reply:
I've added this project as a replacement for 9,10,11 versions.
The tests are empty for those projects (https://github.com/natemcmaster/DotNetCorePlugins/blob/v2/test/TestProjects/JsonNet11/Class1.cs), so this can be added in the future.
The idea was to remove all libraries that have vulnerabilities

Reply:
Ok. These old versions were selected intentionally because this loader is trying to assert that it can load different major versions of an assembly. As this is test-only code, I am not concerned about security vulnerabilities.